cfme/common/provider.py

```diff
@@ -167,6 +167,14 @@ def get_yaml_data(self):
             raise ProviderHasNoKey(
                 'Provider {} has no key, so cannot get yaml data'.format(self.name))
 
+    def has_nodes(self):
```

Comment:
The logic here is that the 'node' label could change to set as 'hosts', in case it has different label and Exception will be thrown